src/main/java/io/quarkus/gizmo/Switch.java

```diff
@@ -21,40 +21,36 @@ public interface Switch<T> {
      * <p>
      * For example, if fall through is disabled then:
      * 
-     * <pre>
-     * <code>
+     * <pre>{@code
```

Comment:
What's the benefit of `<pre>{@code`? Because readability it is not...

Reply:
It actually is way more readable in IntelliJ, but I confess I didn't try Eclipse :-)

I'll revert these changes, as they are not relevant to the present PR.

Reply:
Done.

Reply:
Well, I was not talking about the rendered javadoc but pure text editor :shrimp:

Reply:
Ah OK. I always prioritize the rendered form over the source text, but I agree the source is ugly. Unfortunately, `{@snippet ...}` was only added in JDK 18 and the [Codesnippet Doclet](https://github.com/jtulach/codesnippet4javadoc) is GPL3, so I'm hesitant to use it anywhere.